src/visualBuilder/visualBuilder.style.ts

```diff
@@ -504,7 +514,7 @@ export function visualBuilderStyles() {
             outline: 2px solid #bd59fa;
         `,
         "visual-builder__pseudo-editable-element": css`
-            z-index: 2147483647 !important;
+            z-index: 99999 !important;
```

Comment:
```suggestion
            z-index: 2147483640 !important;
```

Reply:
not working over 99999. tested with this value overall